.travis.yml

```diff
@@ -9,16 +9,24 @@ php:
   - hhvm
   - nightly
 
+env:
+- NO_STYLE=1
+
 cache:
   directories:
     - $HOME/.composer/cache
 
 matrix:
+  include:
+    - php: 7.1
```

Comment:
This adds an extra php7.1 test that includes the style test – it's removed from composer.json by default in the way it's set out (so no failure)